frontend/src/lib/components/icon/Icon.type.ts

```diff
@@ -0,0 +1,24 @@
+import type {SvelteHTMLElements} from 'svelte/elements';
+import type {Color} from '$lib/components/shared/colors';
+import type {IconName} from './icons';
+
+export type IconProps = SvelteHTMLElements['div'] & {
```

Comment:
Should this be `SvelteHTMLElements['svg']`

Reply:
Good catch! I changed this so many times, I forgot to change it here :D